src/lib/components/StatusFilter.svelte

```
@@ -0,0 +1,68 @@
+	const label = $derived(value === 'true' ? 'Active' : value === 'false' ? 'Inactive' : 'Status');
+
+	function selectStatus(selected: string) {
+		const url = new URL(page.url);
+		if (value === selected) {
+			url.searchParams.delete('isActive');
+		} else {
+			url.searchParams.set('isActive', selected);
+		}
+		url.searchParams.set('page', '1');
+		open = false;
+		const target = useResolve ? resolve((url.pathname + url.search) as any) : url;
```

ESLint

[error] Unexpected any. Specify a different type.

(`@typescript-eslint/no-explicit-any`)

---

[error] Unexpected goto() call without resolve().

(svelte/no-navigation-without-resolve)